src/analytics/ValoraAnalytics.ts

```diff
@@ -90,6 +98,22 @@ class ValoraAnalytics {
     } catch (error) {
       Logger.error(TAG, `Segment setup error: ${error.message}\n`, error)
     }
+
+    try {
+      const { accountAddress } = getCurrentUserTraits(store.getState())
+      const stasigUser = accountAddress
+        ? {
+            userID: accountAddress,
+          }
+        : null
+
+      await Statsig.initialize(STATSIG_API_KEY, stasigUser, {
+        overrideStableID: uniqueID,
```

Comment:
is there a reason you're doing this?

Reply:
I'm curious why it's necessary

Reply:
I am getting an error stating that Statsig is uninitialized when I don't specify the stable ID manually.

Reply:
ok. might be worth adding that as a code comment